torch/_dynamo/variables/tensor.py

```diff
@@ -414,6 +414,24 @@ def try_generic_attr_handling():
             raise NotImplementedError
         return result
 
+    def call_id(self, tx):
+        if not self.source:
+            raise NotImplementedError
```

Comment:
Use `unimplemented` to cause a graph break.

Reply:
Same applies to all the time you are using using `NotImplementedError` below.

Reply:
Done